api/config/cdr_config_preprod.json

```diff
@@ -152,6 +150,45 @@
       "wgsHailStoragePath": "wgs/hail.mt",
       "microarrayHailStoragePath": "microarray/hail.mt",
       "microarrayVcfSingleSampleStoragePath": "microarray/vcf/single_sample/alpha3"
+    },
+    {
+      "cdrVersionId": 10,
+      "name": "All of Us Registered Tier Dataset v7",
+      "isDefault": true,
+      "bigqueryProject": "fc-aou-cdr-preprod",
+      "bigqueryDataset": "R2022Q4R3",
+      "creationTime": "2022-11-15 00:00:00Z",
+      "releaseNumber": 10,
```

Comment:
I don't know what uses the releaseNumber, but seems it has to be incremented

Reply:
good catch!